docs/en/observability/cloud-monitoring/azure/monitor-azure-openai.asciidoc

```diff
@@ -0,0 +1,337 @@
+. Enter a display name for your app (for example, `elastic-agent`).
+. Specify who can use the app.
+. The {agent} doesn't use a redirect URI, so you can leave this field blank.
+. Click **Register**.
+. Make note of the **Application (client) ID** because you'll use it to specify the **Client ID** in the integration settings in <<azure-openai-configure-integration>>.
+
+[discrete]
+[[azure-openai-app-credentials]]
+==== Create credentials and add them to your app
+
+Credentials allow your app to access Azure APIs and authenticate itself, so you won't need to do anything at runtime.
+The Elastic Azure OpenAI integration uses client secrets to authenticate.
+
+To create and add client secrets:
+
+. From the https://portal.azure.com/[Azure portal], select the app you created in the previous section.
+. Select **Certificates & secrets** → **Client secrets** → **New client secret**.
+. Add a description (for example, "{agent} client secrets").
+. Select an expiration or specify a custom lifetime.
+. Select **Add**.
+. Make note of the **Value** in the **Client secrets** table because you'll use it to specify the **Client Secret** in <<azure-openai-configure-integration>>.
++
+WARNING: The secret value is not viewable after you leave this page. Record the value in a safe place.
+
+[discrete]
+[[azure-openai-app-role-assignment]]
+==== Add role assignment to your app
+
+To add a role assignment to your app:
+
+. From the https://portal.azure.com/[Azure portal], search for and select **Subscriptions**.
+. Select the subscription to assign the app.
+. Select **Access control (IAM)**.
+. Select **Add** → **Add role assignment**.
+. In the **Role** tab, search for and select **Monitoring Reader**.
+. Click **Next** to open the **Members** tab.
+. Select **Assign access to** → **User, group, or service principal**, and select **Select members**.
+. Search for and select your app name (for example, "elastic-agent").
+. Click **Select**.
+. Click **Review + assign**.
+. Make note of the **Subscription ID** and **Tenant ID** from your Microsoft Entra because you'll use these to specify settings in the integration.
+
+[discrete]
+[[azure-openai-configure-integration]]
+=== Step 4: Configure the Azure OpenAI integration
+
+. Go to the {kib} homepage and click **Add integrations**.
+. In the query bar, search for **Azure OpenAI** and select the Azure OpenAI integration card.
+. Click **Add Azure OpenAI**.
+. Under Integration settings, configure the integration name and optionally add a description.
++
+TIP: If you don't have options for configuring the integration, you're probably in a workflow designed for new deployments.
+Follow the steps, then return to this tutorial when you're ready to configure the integration.
+
+[discrete]
+[[azure-openai-configure-integration-logs]]
+==== Configure logs collection
+
+To collect Azure OpenAI logs, turn on **Collect Azure OpenAI logs from Event Hub**, and specify values for the following required fields:
```

Comment:
The data collection is enabled for both logs and metrics by default now. Do you think we need to include `turn on` flag?